### DISCUSSION TITLE
Starting cache early

### DISCUSSION THREAD

**Discussion by Foixa** [onevcat/Kingfisher#2121] [Q&A]
I want to initialize the cache early even if I don't call an image with Kingfisher. Cache is constantly checking files and deleting the expired file. Is there a way to do this in appdelegate?

**Comment by onevcat** [comment]
Not quite sure what exactly you want to do. But if you want to access the cache, you can do it anywhere by getting the `ImageCache.default`, which is the cache instance behind the shared Kingfisher manager and all the extension methods on image view types.
